### DISCUSSION TITLE
Does `supabase diff --local` detect changes to Table metadata?

### DISCUSSION THREAD

**Discussion by anandkkpr** [supabase/supabase#28777] [Questions] [answered]
I've updated a DB table on my locally running Supabase instance by simply adding a value for the "Description" field using the locally running Supabase Studio UI.

I then used `supabase db diff --local` to see if this change was detected and it was not.

I had thought that these types of changes to the PG DB would be detected.

I then also changed the RLS checkbox setting from "off" to "on" and ran `supabase db diff --local` and, YES, this change WAS detected.

Could someone please let me know:

* Is it expected behaviour that `supabase diff` **does not** track changes to Table metadata - more specifically, the `Description` field?
* Is there a way to have `supabase diff / supabase pull --local` detect this change?

Thank you!

**Comment by GaryAustin1** [comment, marked as answer]
https://github.com/orgs/supabase/discussions/19478
It is a limitation, same with at least column grants.

**Reply by anandkkpr** [reply]
Thank you for this @GaryAustin1 !
